Classes/Eel/ElasticSearchQueryBuilder.php

```diff
@@ -131,6 +131,55 @@ public function nodeType(string $nodeType): QueryBuilderInterface
         return $this->queryFilter('term', ['neos_type_and_supertypes' => $nodeType]);
     }
 
+    /**
+     * Filter multiple node types
+     *
+     * @param array $expectedNodeTypes NodeTypes that should be expected
+     * @param array $excludedNodeTypes NodeTypes that should be excluded
+     * @return ElasticSearchQueryBuilder
+     * @throws QueryBuildingException
+     * @api
+     */
+    public function nodeTypeFilter(array $expectedNodeTypes, array $excludedNodeTypes): QueryBuilderInterface
```

Comment:
Would be nice if you would add a slim test in `ElasticSearchQueryTest`.

Reply:
I added a test and adjusted the existing tests to match the new circumstances.